test/index.ts

```diff
@@ -138,6 +138,31 @@ describe('.hasExpired()', () => {
   });
 });
 
+describe('.isTokenExpiring()', () => {
+  it('should exist', () => {
+    const gtoken = new GoogleToken();
+    assert.strictEqual(typeof gtoken.isTokenExpiring, 'function');
+  });
+
+  it('should detect expiring tokens', () => {
+    const gtoken = new GoogleToken();
```

Comment:
For ease of migrating this behavior from the auth library to the token library, I think we should make the functionality disabled by default.